boilerplates/be-python-flask/README.md

```diff
@@ -6,7 +6,26 @@ Use this quickstarter when you want to build a [python](http://flask.pocoo.org/d
 It contains the basic setup for [Docker](https://www.docker.com/), [Jenkins](https://jenkins.io/), [SonarQube](https://www.sonarqube.org/) and [OpenShift](https://www.openshift.com/).
 
 ## What files / architecture is generated?
-
+```
+.
+├── Jenkinsfile  -  This file contains Jenkins build configuration settings
+├── README.md  -  Readme file template
+├── docker-compose.yml  -  For running all services locally
+├── jenkinsfile_helper.py  -  Helper function used during the Jenkins pipeline
+├── build.sh  -  Build Script. This script copies all necessary files to the docker images 
+├── docker-prediction  -  This folder contains Docker configuration settings for prediction service
```

Comment:
this is not the right file/folder structure, this is the one related to ds-ml-service
@hugowschneider let me know if you want me to update it.